### ISSUE TITLE
help people request TestRail access

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-team#50590]
### Description

As a new user to TestRail, I need to easily find how to request an account so that I can better engage with correct processes.

The TestRail login screen says this:

> Please visit the [VA.gov Platform Website](https://depo-platform-documentation.scrollhelp.site/developer-docs/How-to-use-TestRail.1600684126.html) to learn how to get an account as well as some best practices for working within TestRail.

However, it landed me on a [table of contents](https://depo-platform-documentation.scrollhelp.site/developer-docs/qa-and-accessibility-testing) that did not help me find how to request an account. Nor did any content I could find within any of the pages.

I still haven't figured it out so please LMK. Thank you

**Comment by vbellissimo** [comment]
@wesrowe thanks for submitting this. Can you share a URL or a screenshot of the TestRail login screen?

**Comment by wesrowe** [comment]
sure:

<img width="466" alt="image" src="https://user-images.githubusercontent.com/4054752/206804410-dbd8ae4e-23b4-4c94-a4ec-beeabc616179.png">

**Comment by vbellissimo** [comment]
@wesrowe Thank you. We will be reviewing this for next steps.

**Comment by wesrowe** [comment]
@vbellissimo, in the meantime, how do I request access to TestRail? I looked for pinned items in #platform-qa-standards in slack and didn't see anything either.

**Comment by vbellissimo** [comment]
@wesrowe I asked around and this is what I've been told:
_If you do not have a TestRail account, request help in #vfs-platform support._

We'll work on updating that login screen and some of the verbiage on the Platform website around this.

**Comment by vbellissimo** [comment]
@wesrowe Hi Wes, just wanted to let you know that we have made a few updates to the content on the Platform website, [including this page](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-project-in-testrail), with the instructions we gave you to seek support from Slack for a TestRail account. We also asked for someone to update the hyperlink on the TestRail login page you pointed out to go to this same page. Let us know if you have any other questions. Thank you for alerting us on this.
